### DISCUSSION TITLE
HELP!!!! Failed to resolve import "src/lib/utils" from ... Does the file exist?

### DISCUSSION THREAD

**Discussion by tmhourani** [shadcn-ui/ui#3897] [General]
Hello everyone, 
I am trying to follow this tutorial (https://www.youtube.com/watch?v=_W3R2VwRyF4) on youtube and am trying to test out the button component, but I keep getting an error message: 

[plugin:vite:import-analysis] Failed to resolve import "src/lib/utils" from "src/components/ui/button.tsx". Does the file exist?

Both "utils.ts" and "button.tsx" are located in the correct path, so I am really confused why it is not working. 

Here is a screen shot of the rest of the error message:

![Screenshot_2-6-2024_102432_localhost](https://github.com/shadcn-ui/ui/assets/170745251/2ef9913d-58ba-4193-a388-1b9ee537354c)

I am assuming that it is a syntax OR a path error. Does anyone have any thoughts? Thank you.

**Comment by abhayclasher** [comment]
vite doesn't resolve absolute paths like `src/lib/utils` by default. just change the import in your `button.tsx` to a relative path like `../../lib/utils` or set up the `@` alias in your `vite.config.ts` and use `@/lib/utils` instead. running `npx shadcn-ui@latest init` usually configures this for u automatically.
